projects/experimental/directives/card/card.styles.less

```diff
@@ -0,0 +1,57 @@
+@import 'taiga-ui-local';
+
+[tuiCard] {
+    .clearbtn();
+    display: flex;
+    background: var(--tui-base-02);
+    border-radius: 0.75rem;
+    box-sizing: border-box;
+    padding: 0.75rem;
+    gap: 0.75rem;
+    flex-direction: column;
+    text-decoration: none;
+
+    &[data-size='m'] {
+        width: 8.75rem;
+        height: 8.75rem;
+    }
+
+    &:not([data-platform='android']) {
+        transition: transform 100ms ease-in;
+    }
+
+    &._touched[data-platform='ios'] {
+        transform: scale(0.95);
+    }
+
+    &[data-platform='web']:hover {
+        transform: scale(1.15);
+    }
+
+    & > * {
+        overflow: hidden;
+    }
+
+    & > :nth-child(2),
+    [tuiCardBottom] {
```

Comment:
Let's just add `[style.margin-top]="'auto'"` to the example. It takes single line of code, just like adding `tuiCardBottom`, but does not expand our public API.

Reply:
fixed